### DISCUSSION TITLE
macOS version 3.6.0?

### DISCUSSION THREAD

**Discussion by schusz** [Jacalz/rymdport#157] [General]
I have downloaded rymdport.exe 3.6.0 for Windows, but am missing 3.6.0 for macOS.
Will there be a 3.6.0 for macOS?

**Comment by Jacalz** [comment]
Thanks for bringing this to my attention. Sorry about that. I have automatic cross compiled builds set up using https://fynelabs.com/geoffrey/ and the macOS builds failed due to some problem on their side. I had a few hectic days and forgot about it. Will kick the build again and notify them if the issue persists. Will get binaries uploaded as soon as possible.

**Reply by Jacalz** [reply]
New binaries are uploaded and automatic updates have been pushed :)
